src/Components/Seo/ArtistStructuredData.tsx

```diff
@@ -4,40 +4,22 @@ import type { ArtistStructuredData_artist$key } from "__generated__/ArtistStruct
 import compact from "lodash/compact"
 import { graphql, useFragment } from "react-relay"
 
-interface Props {
+interface ArtistStructuredDataProps {
   artist: ArtistStructuredData_artist$key
 }
 
-export const ArtistStructuredData: React.FC<Props> = ({ artist }) => {
+export const ArtistStructuredData: React.FC<ArtistStructuredDataProps> = ({
+  artist,
+}) => {
   const data = useFragment(fragment, artist)
   const artistUrl = `${getENV("APP_URL")}${data.href}`
 
-  const creatorOf = data.artworks_connection?.edges
-    ?.map(edge => {
-      const artwork = edge?.node
-      if (!artwork || !artwork.title || !artwork.href) return null
-      const artworkUrl = `${getENV("APP_URL")}${artwork.href}`
-
-      return {
-        "@type": "VisualArtwork",
-        name: artwork.title,
-        dateCreated: artwork.date,
-        artform: artwork.category,
-        url: artworkUrl,
-        image: artwork.image?.large,
-      }
-    })
-    .filter(Boolean)
-
+  // FIXME: This isn't strictly correct and does not encapsulate representation; also wouldn't be unique
```

Comment:
So yeah, Botify is incorrect about what they specified.

![](https://capture.static.damonzucconi.com/Screen-Shot-2025-05-20-08-32-03.79-6CmxcshMdCOpF2v2ZEyErC6TtzR3AeiLkhtsxw2J68EqsFC1C2wx54bio5oar8C1hMuMljLEFyeYwmux9TDa3xZOBeEwKwvqSP2P.png)

See https://schema.org/memberOf